src/languages/es.js

```diff
@@ -734,6 +734,14 @@ export default {
             chatWithConcierge: 'Chat con Concierge',
             letsFinishInChat: '¡Acabemos en el chat!',
             almostDone: '¡Casi listo!',
+            disconnectBankAccount: 'Desconectar cuenta bancaria',
+            noLetsStartOver: 'No, empecemos de nuevo',
+            startOver: 'Empezar de nuevo',
+            yesDisconnectMyBankAccount: 'Sí, desconecta mi cuenta bancaria',
+            yesStartOver: 'Sí, empezar de nuevo',
+            disconnectYour: 'Desconecta tu cuenta bancaria de ',
+            bankAccountAnyTransactions: '. Lost reembolsos pendientes serán completados sin problemas.',
```

Comment:
There seems to be a typo here: 
```suggestion
            bankAccountAnyTransactions: '. Los reembolsos pendientes serán completados sin problemas.',
```